web/typescript/main.ts

```diff
@@ -51,6 +52,10 @@ function pageSpecificFunctions(settings: UserSettings) {
     if (location.pathname.startsWith('/style/') && styleViewRegex.test(location.pathname)) {
         initViewStyle();
     }
+
+    if (location.pathname.startsWith('/search') || location.pathname.startsWith('/explore') || location.pathname.startsWith('/user/')) {
```

Comment:
```suggestion
    if (location.pathname.startsWith('/search') || location.pathname.startsWith('/explore') || location.pathname.startsWith('/user')) {
```

Reply:
There's not so much difference, but i've added the `/` because of the `startsWith('/style/')` above to follow the common code style.